### PR TITLE
Dami/add ci

### LINKED ISSUE #41: Include Itegration tests action into CI
Integration tests should run for every PR opened in CI

- [x] Ensure The contracts Build In CI

- [x]  Ensure the Substrate Contract Node runs in CI

- [x] Ensure Integration Tests runs in CI

### PR DESCRIPTION
**Pull request by dharjeezy**
closes #41

### CODE REVIEW

File: tests/util.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import {ChildProcess, spawn} from 'child_process';
+
+const substrateContractNodePath = './artifacts/substrate-contracts-node-linux/substrate-contracts-node';
+export async function startContractNode() {
+    const startArgs: string[] = [];
```
Comment (drewstone) [inline tests/util.ts:5]: Tabs in typescript should be 2 spaced.

File: tests/util.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import {ChildProcess, spawn} from 'child_process';
```
Comment (drewstone) [inline tests/util.ts:1]: Please add spaces to this imports
```suggestion
import { ChildProcess, spawn } from 'child_process';
```

File: tests/tokenWrapper.test.ts [hunk 3]
```diff
@@ -3,6 +3,8 @@ import { artifacts, network, patract } from "redspot";
 import BN from "bn.js";
 import { all } from "@polkadot/api-derive/balances";
 import { hexToU8a } from "@polkadot/util";
+import {ChildProcess, spawn} from 'child_process';
```
Comment (drewstone) [inline tests/tokenWrapper.test.ts:6]: Please add spaces to imports 
```suggestion
import { ChildProcess, spawn } from 'child_process';
```

File: tests/tokenWrapper.test.ts [hunk 4]
```diff
@@ -3,6 +3,8 @@ import { artifacts, network, patract } from "redspot";
...
+import {startContractNode} from "./util";
```
Comment (drewstone) [inline tests/tokenWrapper.test.ts:7]:
```suggestion
import { startContractNode } from "./util";
```

**Review by drewstone — CHANGES_REQUESTED** [review]
Can you fix up the commit history since lots of changes here seem to have been applied in previous PRs?

**Comment by dharjeezy** [comment]
closing this in honour of this #44 
the commit history is wrong

**Pull request closed without merging**